docker/entrypoint.sh

```diff
@@ -117,7 +117,9 @@ fi
 
 
 if [[ "$*" == "" ]]; then
-    CMD=("run" "-p" "${PYPISERVER_PORT:-$PORT}")
+    # Use the gunicorn server by default, since it's more performant than
+    # bottle's default server
+    CMD=("run" "-p" "${PYPISERVER_PORT:-$PORT}" "--server" "gunicorn")
```

Comment:
I know it's not technically part of this PR, but should we do a check here to see if `$PORT` is and give a deprecation warning if it is, before using it? (I guess we then also need to not set it in `Dockerfile`)

Reply:
I struggled a bit with this, because what if they have the env var `$PORT` set for some other reason or as part of something else they've got running on a derived docker container?

Reply:
Hmm, I see. What about not showing that warning when a $PYPISERVER_DISABLE_DEPRECATION_WARNINGS variable is set? or is that taking things too far :sweat_smile: ?

Reply:
Hahaha, I think that's probably good actually, although then I'd have to go update the deprecation warning I'm showing for the old arg form as well

Reply:
Yeah exactly. One would expect that to not only hide docker-specific deprecations, but also those coming from the python source

Reply:
I would like to think a bit more about this and potentially do it in a separate PR, but I do think it's a good idea